packages/code/src/renderingPlugin.tsx

```diff
@@ -87,6 +101,9 @@ function getHighlightedRowStyle(darkMode: boolean) {
   return css`
     background-color: ${backgroundColor};
     background-image: ${backgroundImage};
+    // Fixes an issue in Safari where the gradient applied to the table row would be applied
+    // to each cell in the row instead of being continuous across cells.
```

Comment:
I'm still seeing a different highlight color in darkmode across Chrome and Safari.

Chrome:
![Screen Shot 2021-01-04 at 10 43 42 AM](https://user-images.githubusercontent.com/26016393/103552406-dd8dd000-4e79-11eb-884b-1a3dd8157e65.png)

Safari:
![Screen Shot 2021-01-04 at 10 43 45 AM](https://user-images.githubusercontent.com/26016393/103552442-e9799200-4e79-11eb-8ddf-4d493e15dc0a.png)

Reply:
That's a difference in the way gradients render across browsers, not a bug in the implementation here.

Reply:
Is there a way to make them look more visually consistent, even if it isn't a bug in implementation?

Reply:
Hah I spent a little more time with this problem and it brought be back a bit to a different cross-browser issue I found with gradients probably 4 years ago. Transparent in Safari is `rgba(0, 0, 0, 0)`, but transparent in Chrome is `rgba(255, 255, 255, 0)`. Usually, this is not a meaningful distinction, but in gradients it means that the second stop would be lighter than it should in Chrome, and darker in Safari.

I'm pushing a fix for this now. It's worth noting that there are still issues with colors rendering differently across browsers that's good to be aware of, but this was a different case.

Reply:
Looks a lot better!